core/domain/subscription_services.py

```diff
@@ -57,14 +57,14 @@ def subscribe_to_activity(user_id, activity_id):
     if not subscriptions_model:
         subscriptions_model = user_models.UserSubscriptionsModel(id=user_id)
 
-    if (activity_id not in
-            subscriptions_model.activity_ids):
+    if activity_id not in subscriptions_model.activity_ids:
         subscriptions_model.activity_ids.append(activity_id)
         subscriptions_model.put()
 
 
 def get_activity_ids_subscribed_to(user_id):
-    """Returns a list with all subscriptions' ids of given user.
+    """Returns a list with ids of all activities (i.e., explorations and
```

Comment:
Is this just 'explorations'? (See [old comment](https://code.google.com/p/oppia/source/diff?spec=svn5c2606468772bf7249a41f3d65d38443c5de2b39&name=collections-backend&r=5c2606468772bf7249a41f3d65d38443c5de2b39&format=side&path=/core/domain/subscription_services.py) for get_collection_ids_subscribed_to() -- just marking this as needing to be resolved somehow.